tools/osbuild-image-test

```diff
@@ -400,7 +400,7 @@ def run_tests_cases(args):
     success = True
     if not tests:
         print(f"{RED}FAIL{RESET}: no tests to run")
-        return False
+        return 1
```

Comment:
Sure why not, looks better, feel free to commit this :) .. I'm mainly concerned with the fact, that the CI job passes despite logs showing failure because no tests were collected to be executed.

Reply:
Thanks! I will do followup then :) Totally fine to merge as is as it's ready

Reply:
Yeah, I just need to figure out how to add the missing test cases :sweat_smile:

Reply:
This software is going to be decommissioned soon, you're very welcome to hack it as much as you want, just know that the result is not going to live long :slightly_smiling_face:

Reply:
I suppose this change is not needed after all, marking as resolved.